regenmaschine/client.py

```diff
@@ -154,8 +147,12 @@ async def _request_with_session(
             RequestError: Raised upon an underlying HTTP error.
         """
         try:
-            async with timeout_callable(self._request_timeout), session.request(
-                method, url, ssl=self._ssl_context if use_ssl else None, **kwargs
+            async with session.request(
+                method,
+                url,
+                ssl=self._ssl_context if use_ssl else None,
+                timeout=self._request_timeout,
+                **kwargs,
             ) as resp:
                 data = await resp.json(content_type=None)
```

Comment:
`json` will still raise TimeoutError if the timer expires https://github.com/aio-libs/aiohttp/blob/43f3e23b1c66ad8303fd1333ccb86c2301d07d15/aiohttp/streams.py#L301